js/components/Header.js

```diff
@@ -53,10 +54,26 @@ const Header = React.createClass ({
   },
 
   /* Methods */
-  navigateBack() {
+  backToLogin() {
+    this.setState({title: ''})
     this.props.navigator.pop()
```

Comment:
I'm not sure if it could ever happen, but we could call on the navigator.resetTo({path:'login',title:''}) just to be absolutely sure that we are going to the login page and keeping the navigation stack clean. Thoughts?